DB/main.py

```diff
@@ -0,0 +1,30 @@
+import firebase_handler as fh  # Assuming your previous code was saved in firebase_handler.py
+
+
+def main():
```

Comment:
Adding docstrings to your functions can greatly improve the readability of your code and help other developers understand the purpose of each function.